src/components/docs/MermaidWrapper/MermaidWrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from './MermaidWrapper.module.css'
+import { useRef } from 'react'
+
+export default function MermaidWrapper({ children }) {
+  const wrapperRef = useRef(null)
+
+  const clickHandler = () => {
+    const svg = wrapperRef.current.querySelector('svg')
```

Comment:
A bit of a hack to get the Mermaid-generated SVG to display in a new tab since it's inline and not a file I can reference via href